.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1'
+      - name: Install dependencies
+        run: julia --project=docs/ -e 'using Pkg;
+                Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+      - name: Build and deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # For authentication with GitHub Actions token
+          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }} # For authentication with SSH deploy key
```

Comment:
Same comment as above

Reply:
According to Documenter.jl [docs](https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#GitHub-Actions) it seems that in this case it should be fine as it is now. Maybe we can try to see how it works, and fix it if necessary

Reply:
Just check, and  this comment does not apply... Sorry.